src/components/ens-registration/TextRecordsForm/SelectableButton.tsx

```diff
@@ -11,37 +11,42 @@ type SelectableButtonProps = {
   children: ReactNode;
   onSelect: () => void;
   isSelected: boolean;
+  testID?: string;
 };
 
 export default function SelectableButton({
   children,
   onSelect,
   isSelected,
+  testID,
 }: SelectableButtonProps) {
+  const secondary06 = useForegroundColor('secondary06');
   const secondary30 = useForegroundColor('secondary30');
   const accent = useForegroundColor('accent');
-  const buttonColor = isSelected ? accent : secondary30;
+  const borderColor = isSelected ? accent : secondary06;
+  const textColor = isSelected ? accent : secondary30;
   const height = 30;
 
   return (
-    <AccentColorProvider color={buttonColor}>
+    <AccentColorProvider color={textColor}>
       <Box
         as={ButtonPressAnimation}
         height={`${height}px`}
         // @ts-expect-error
         onPress={onSelect}
+        testID={testID}
       >
         <Box
           alignItems="center"
           borderRadius={12}
           height={`${height}px`}
           justifyContent="center"
           paddingHorizontal="8px"
-          style={useMemo(() => ({ borderColor: buttonColor, borderWidth: 2 }), [
-            buttonColor,
+          style={useMemo(() => ({ borderColor: borderColor, borderWidth: 2 }), [
+            borderColor,
           ])}
```

Comment:
I cannot argue that, but I guess we are OK with rendering primitives like `Box` and we should not memorize here (at least we don't do this anywhere in the codebase)

Reply:
yeah agree, will remove